tests/test_benchmark_django.py

```diff
@@ -0,0 +1,6447 @@
+# NOTE: This file is used for benchmarking. Before editing this file,
```

Comment:
I've updated the files with the benchmark code to refer back to this PR, so people have a chance to read up on all these nuances if they need to make any changes to the files in the future.